scripts/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -ex
+VALUES_FILE=$(pwd)/values.yaml
+
+PRIVATE_SPHINX=$(cat $PRIVATE_KEY_CONTENT | base64 | tr -d '\n')
+PUBLIC_SPHINX=$(cat $PUBLIC_KEY_CONTENT | base64 | tr -d '\n')
+
+cat > $VALUES_FILE <<EOF
+
+environment: production
+nym:
+  layer: 3
+  location: europe-west3
+  data:
+    private_sphinx: |
+      $PRIVATE_SPHINX
+    public_sphinx: |
+      $PUBLIC_SPHINX
+EOF
+
+cat ${VALUES_FILE}
+
+/scripts/deploy.sh -t helm -c engineering -a "nym-mixnode w3f/nym-mixnode -f ${VALUES_FILE}"
```

Comment:
In these cases where you need to create a values file I think it would be much simpler and straightforward to use helmfile from the beginning, wdyt?